packages/docs/docs/guides/3-API/3-api-apps.md

```diff
@@ -4,10 +4,37 @@ keywords: [Request, Apps, API]
 
 ---
 
-# About Apps
+# Introduction
 
-TODO: Here explain why: manage requests of your users
+Developers can build on top of Request API by creating Applications.
+
+With an application, you will be able to read, create, and perform actions on requests, on behalf of your users. 
 
 # How it works
 
-TODO: Get in touch with us
+An application built on top of Request API needs to authenticate with the OAuth2 mechanism.
+We recommend using [Auth0 SDKs](https://auth0.com/docs/libraries#sdks) to integrate it, but you are free to choose whatever you want.
+
+# Getting started
+
+TODO: unlock the Apps feature for all users?? 
```

Comment:
What do you mean "for all users" ?

Reply:
currently, the Apps section of the settings is behind a user-based feature flag, handled by auth0. If we add this to the docs, we might want to open the feature to all.